### PR TITLE
add robotframework sdk to the list of supported sdks

### PR DESCRIPTION
**Pull request by Artem-applitools**

### CODE REVIEW

**Review by kyryloonufriiev — APPROVED** [review]
LGTM

File: pending-changes.yaml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+"@applitools/sdk-coverage-tests":
+  feature:
+    - Support reporting for the robotframework sdk
+  bug-fix:
+    -
```
Comment (kyryloonufriiev) [inline pending-changes.yaml:5]: @Artem-applitools Please remove it. There is no need for a changelog for something else than SDK. This is an internal tooling package so we don't maintain changelog for them

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]

File: pending-changes.yaml (hunk 1 above)
Comment (Artem-applitools) [reply pending-changes.yaml]: @kyryloonufriiev Removed

**Comment by Artem-applitools** [comment]
@kyryloonufriiev Pls merge the PR when you will be publishing new package version

**Pull request merged**